src/Agent.Worker/ContainerOperationProvider.cs

```diff
@@ -393,15 +404,6 @@ public async Task StopContainerAsync(IExecutionContext executionContext, object
                 {
                     executionContext.Error($"Docker rm fail with exit code {rmExitCode}");
```

Comment:
let's change this to warning from error, since all deletion are best effort, same comment for network delete.

Reply:
Done